ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/schema_fragments/router_bgp.schema.yml

```
@@ -1331,6 +1342,15 @@ keys:
                     Example: MyFunction(myarg).
                     `route_map` and `rcf` are mutually exclusive. `route_map` takes precedence.
                     Only applicable if `source_protocol` is one of `connected`, `static`, `isis`, `user`, `dynamic`.
+                ospf_route_type:
+                  type: str
+                  valid_values:
+                    - "external"
+                    - "internal"
+                    - "nssa-external"
+                    - "nssa-external 1"
+                    - "nssa-external 2"
+                  description: The `ospf_route_type` is required with source_protocols 'ospf' and 'ospfv3'.
```

same as above comment and we can add the description as
 ```suggestion
                  description: |
                  Routes learned by the OSPF protocol.
                  The `ospf_route_type` is valid with source_protocols 'ospf' and 'ospfv3'.
```

---

Changed it!